src/content/peerconnection/restart-ice/js/test.js

```diff
@@ -52,12 +50,10 @@ test('PeerConnection restart ICE sample', {skip: true}, t => {
       t.pass('got media');
       return driver.wait(() => driver.findElement(webdriver.By.id('callButton')).isEnabled(), 30 * 1000);
     })
+    .then(() => driver.findElement(webdriver.By.id('callButton')).click())
     .then(() => {
-      driver.findElement(webdriver.By.id('callButton')).click();
       t.pass('Pressed callButton');
-      return driver.wait(() => {
-        return driver.executeScript('return pc1 && pc1.iceConnectionState === \'completed\' || \'connected\';');
-      }, 30 * 1000);
+      return driver.wait(() => driver.executeScript(() => window.pc1 && (pc1.iceConnectionState === 'completed' || pc1.iceConnectionState === 'connected')))
```

Comment:
the string executeScripts were really ugly -- and potentially hiding js errors from a linter